bee-api/bee-rest-api/src/endpoints/mod.rs

```diff
@@ -4,3 +4,47 @@
 pub mod api;
 pub mod debug;
 pub mod health;
+
+use crate::{config::RestApiConfig, storage::StorageBackend, Bech32Hrp, NetworkId};
+
+use bee_network::NetworkServiceController;
+use bee_protocol::{config::ProtocolConfig, MessageSubmitterWorkerEvent, PeerManager};
+use bee_runtime::{node::NodeInfo, resource::ResourceHandle};
+use bee_tangle::MsTangle;
+
+use tokio::sync::mpsc;
+use warp::{self, Filter, Rejection, Reply};
+
+use std::net::IpAddr;
+
+pub(crate) fn filter_all<B: StorageBackend>(
+    public_routes: Vec<String>,
```

Comment:
The changes look good to me. Great job!
Another thing we could refactor in the future:
Instead of passing all the arguments to all filters like that we could define a single "Options" object which contains all the types. And we could wrap them into `Arc`s.
But yeah this is something for the future.

Reply:
Thanks! I agree, that would make things nice and clean later on.